app/javascript/src/components/Dashboard/Notes/NotesHeader.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+
+import { Button } from "neetoui";
+import { Header } from "neetoui/layouts";
+
+const NotesHeader = ({ setShowNewNotePane }) => {
```

Comment:
https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing

Reply:
Made the changes.